examples/gen-ai-hello-world/.env.example

```
@@ -0,0 +1,2 @@
+OPENAI_API_KEY = "<YOUR_OPENAI_API_KEY>"
```

Ini di antara valuenya perlu petik kah?

```suggestion
OPENAI_API_KEY = <YOUR_OPENAI_API_KEY>
```

---

ah sorry, yang ini terlewat, aku update lagi

---

tanda petik di remove di sini [aefbb73](https://github.com/GDP-ADMIN/gen-ai-examples/pull/1/commits/aefbb737387f38ad690c35babf94a4a03695f1e4)